src/util/Sudoku_Generator.py

```diff
@@ -0,0 +1,72 @@
+import random
+import sys
+sys.path.append("../algorithms")
+from Brute_Force import BruteForce
+
+
+class SudokuGenerator(object):
+	"""Generates a sudoku game depends of the complexity level sent"""
+	def __init__(self):		
+		# EMPTY_SUDOKU - string, is the empty sudoku going to send to 
+		self.EMPTY_SUDOKU = '000000000000000000000000000000000000000000000000000000000000000000000000000000000'
+	
+	def generate_random_list(self, number, list):
+		"""
+		Generates a random list to mask cells and display zero instead of cell's 
+		solution.
+		number - int, number of cells per row to be maintained and not modified.
+		list - int, list to modify with zero.
+		Return - list numbers
+		"""
+		index = random.randint(0, len(list) - 1)
+		result = [list.pop(index)]
+		if(number > 1):
+			result.extend(self.generate_random_list(number - 1, list))
+		
+		return result
+
+	def get_minimum_replaced_cells_with_zero(self, dificult):
+		"""
+		Gets the minimum number of cells going to be replaced by Zero in a 
+		Sudoku grid.
+		dificult - string, sudoku's dificult sent
+		Return - returns the minimum cant of cells going to be masked and
+		displayed as zero
+		"""
+		minimum_replaced_cells_with_zero = 5
+		
+		if dificult == "easy":
+		    minimum_replaced_cells_with_zero = 3
```

Comment:
these values should be constants or read from the config file if you have this configured there